light_curves/multiband_lc.ipynb

```diff
@@ -110,47 +91,39 @@
     "    !pip install acstools\n",
     "    from acstools import acszpt\n",
     "\n",
-    "import warnings\n",
-    "warnings.filterwarnings('ignore')\n",
+    "try:\n",
+    "    from alerce.core import Alerce\n",
+    "except ImportError:\n",
+    "    !pip install alerce\n",
+    "    from alerce.core import Alerce\n",
     "\n",
     "try:\n",
     "    from unTimely_Catalog_tools import unTimelyCatalogExplorer\n",
     "except ImportError:\n",
-    "    if not os.path.exists('./unTimely_Catalog_explorer'):\n",
-    "        !git clone https://github.com/fkiwy/unTimely_Catalog_explorer.git\n",
-    "    sys.path.append('./unTimely_Catalog_explorer')\n",
+    "    !pip install git+https://github.com/fkiwy/unTimely_Catalog_explorer\n",
+    "    !pip install reproject\n",
     "    from unTimely_Catalog_tools import unTimelyCatalogExplorer\n",
```

Comment:
I suggest chopping up the imports and moving them to the beginning of the different big sections of the notebook. E.g. make it possible to obtain all the data without having any of the exploration tools installed/be ready.